src/rt/sched/behaviourcore.h

```diff
@@ -357,6 +357,126 @@ namespace verona::rt
       body->resolve(ec);
     }
 
+    template<TransferOwnership transfer = NoTransfer>
```

Comment:
There is a lot of code duplication here with the prior version, can we build a single one that does everything.  Do we still need schedule one?